crates/storage/db/src/tables/codecs/postcard.rs

```diff
@@ -36,5 +39,5 @@ macro_rules! impl_postcard {
 
 type VecU8 = Vec<u8>;
 
-#[cfg(feature = "bench-postcard")]
-impl_postcard!(VecU8, Receipt, H256, U256, H160, u8, u16, u64, Header, Account, Log, TxType);
+//#[cfg(feature = "bench-postcard")]
```

Comment:
@joshieDo I removed postcard as it clashes with impl of Decode/Encode from table.

Reply:
Wdym it crashes?

Reply:
`Decode/Encode` are implemented in two places. 
One in tables:
https://github.com/paradigmxyz/reth/blob/e014eaad339e4b01f6c682614a15c0c0651cc218/crates/storage/db/src/tables/models/mod.rs#L19-L46
And second one is for postcard:
https://github.com/paradigmxyz/reth/blob/e014eaad339e4b01f6c682614a15c0c0651cc218/crates/storage/db/src/tables/codecs/postcard.rs#L20-L43

When I thnk about it now, postcard probably needs to implement `Compress/Decompress` and not Encode/Decode.

Reply:
Interesting. Honestly having both of these traits is kinda confusing. @joshieDo ideas on how we can improve this?

Reply:
> When I thnk about it now, postcard probably needs to implement Compress/Decompress and not Encode/Decode.

Hm, yeah when we made the change to `Compress/Decompress`, we didn't change these.